src/apic-extension/azext_apic_extension/tests/latest/test_service_commands.py

```diff
@@ -235,32 +232,3 @@ def test_examples_show_service_details(self):
     @ApicServicePreparer()
     def test_examples_update_service_details(self):
         self.cmd('az apic update -g {rg} -n {s}')
-
-    def _prepare_apim(self):
-        if self.is_live:
```

Comment:
why only prepare apim in live mode

Reply:
In playback mode, the APIC doesn't require APIM to be exist. We just need to ensure the import from APIM request returns mocked response and don't need to really import from APIM, as the import happens in server side not client side.